README.md

```diff
@@ -1,24 +1,86 @@
 # Purdue Laundry API
 
-Backend for the Purdue Laundry application. It scrapes data from the Purdue Laundry alert page and provides a nice API for accessing it from the app. 
+This is a backend for the Purdue Laundry application. It scrapes 
+data from the Purdue Laundry alert page and provides a nice API 
+for accessing it from the app. 
 
+# Running the Server with Docker
 
-## Contributing
+Our server is Dockerized, so running it will require you to install
+`docker` and `docker-compose`. Refer to the [docker documentation](https://docs.docker.com/compose/install/)
+for instructions on how to do that. It's fairly well documented
+and painless. 
 
-Anyone is free to contribute to this server. However attention should be paid to the PurdueLaundry Application for compatability with API changes. 
+Once you have those 2 dependencies installed, you can install dependencies and build and run
+the server by simply running:
 
-`master` represents the API as currently deployed to `laundry-api.sigapp.club`. Updates to this should mirror a redeploy of the server. Do not commit to master directly. Do not merge into master without approval. 
+    $ yarn
+    $ yarn start
```

Comment:
Does this run it inside docker?

Reply:
yes. although i should change `yarn` to `yarn build`